pipelines/upgrade/08-tests.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- name: run tests
+  hosts:
+    - "{{ forklift_server_name }}"
+  become: yes
+  vars_files:
+    - ../vars/upgrade_base.yml
+    - ../vars/repos_staging.yml
+  vars:
+    bats_output_dir: "/root/bats_results_proxy_n-1_upgrade"
+  environment:
+    FOREMAN_EXPECTED_VERSION: "{{ foreman_expected_version | default('') }}"
+    ORG_SUFFIX: "_proxy_n-1_upgrade"
+  roles:
+    - role: forklift_versions
+      scenario: "{{ pipeline_type }}"
+      scenario_os: "{{ pipeline_os }}"
+      scenario_version: "{{ pipeline_version }}"
+      upgrade: True
+    - role: forklift_versions
+      scenario: "{{ pipeline_type }}"
+      scenario_os: "{{ pipeline_os }}"
+      scenario_version: "{{ forklift_upgrade_version_final }}"
+    - role: foreman_client_repositories
+      when: pipeline_type != 'foreman'
```

Comment:
Should we gate this role also on the condition at line 28?

Reply:
It doesn't really hurt to run the repositories role here, and it doesn't take much time either. 

Or do you have other reasons to skip it here?

Reply:
Just time saving -- every second counts when it takes eleventy-hours to run